ChangeLog

```diff
@@ -33,6 +33,7 @@
  * Deprecated classes UserDefinedPair, HistogramPair
  * Removed SensitivityAnalysis class
  * Deprecated SLSQP, LBFGS and NelderMead classes in favor of NLopt class
+ * Deprecated LAR in favor of LARS
 
```

Comment:
you don't exactly deprecate the LAR class, you rename it into LARS.
Question : how to still use LAR until 1.10?